docs/tigem.md

```diff
@@ -0,0 +1,7 @@
+# nf-core/configs: TIGEM configuration
+
+To use, run the pipeline with `-profile tigem`. This will download and launch the tigem.config which has been pre-configured with a setup suitable for the TIGEM personal biocluster.
+
+---
+
+For this pipeline you need a docker container containing the tools needed for the specific pipeline. You can find the docker containers on giusmar docker hub profile. A docker image containing the required software will be downloaded, and converted to a Singularity image before execution of the pipeline.
```

Comment:
Maybe reformulate this a bit - it reads as if the user has to do anything (which they really don't apart from waiting) ;-)

Reply:
I want to use a different docker container for each different pipeline, but it is for my configuration.Exactly, nf-core pipelines set a different docker container for each process automatically, so, basically, the better reformulation is to delete the sentence :)
I ll update the doc with further details for 
`process.executor = 'slurm'
google.zone = 'europe-west1'`